planet/js/helper.js

```diff
@@ -138,8 +138,8 @@ $(document).ready(function() {
     });
     document.getElementById('view-more-chips').addEventListener('click', function (evt) {
         toggleExpandable('morechips','flexchips');
-        toggleText('view-more-chips','View More','View Less');
-		toggleText('view-more-chips','और दिखाएं','छिपाएं');
+        toggleText('view-more-chips','View More','View Fewer');
+		toggleText('view-more-chips','और दिखाएं','कम दिखाएं');
 		toggleText('view-more-chips','Ver más','Ver menos');
 		toggleText('view-more-chips','もっと見る','あまり見ない');
```

Comment:
Translations in Music Blocks are handled centrally via the generation of PO files (instead of manually translating phrases in the 'main' code as you have done, you should make the string translatable via the surrounding the string with _( string ) as Walter mentioned :P). 
As I understand, this 'PO' method is pretty neat, in that it's somewhat simple and yet is very scalable; it also seems to make adding new languages, and their translations, straightforward.